### PR TITLE
Update Provider page for next release.

### PR DESCRIPTION
**Pull request by terrylucas**

### CODE REVIEW

File: packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart [hunk 1]
```diff
@@ -16,8 +15,6 @@ import '../../globals.dart';
 import '../../utils.dart';
 import '../provider_debounce.dart';
 import 'eval.dart';
-// TODO(terry): Used fake interface until 3rd party freezed_annotation fixed.
-import 'fake_freezed_annotation.dart';
```
Comment (jacob314) [inline packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart:20]: why is this removed?

**Review by jacob314 — APPROVED** [review]
lgtm

File: packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart (hunk 1 above)
Comment (terrylucas) [reply packages/devtools_app/lib/src/provider/instance_viewer/instance_providers.dart:20]: There we no references to this library.

**Review by jacob314 — APPROVED** [review]
lgtm
